py-polars/Makefile

```
@@ -84,8 +84,17 @@ doctest: .venv build  ## Run doctests
 	$(VENV_BIN)/python tests/docs/run_doctest.py
 	$(VENV_BIN)/pytest tests/docs/test_user_guide.py -m docs
 
+.PHONY: docs
+docs: .venv build  ## Build Python docs (incremental)
+	@cd docs && $(MAKE) html
```

These are definitely useful - wonder why I never thought about adding them :smile: 

Can we use the built-in Make support for running Make commands in other directories though? Something like:

```suggestion
	@$(MAKE) -s -C docs html
```